srv/admin-service.cds

```diff
@@ -9,6 +9,12 @@ service AdminService @(requires : 'admin') {
 
   entity Authors as projection on my.Authors;
   entity Orders  as select from my.Orders;
+
+  @cds.persistence.skip
+  entity Csv @odata.singleton {
+    data     : LargeBinary @Core.MediaType : dataType;
+    dataType : String @Core.IsMediaType;
```

Comment:
Is there still a reason for this element, as it's value is anyway always expected to be `text/csv`? 
Could we also use `@Core.MediaType: 'text/csv'` on data directly?

Reply:
Yeah, we could. That would make sense, I'll change it.